app/api/generate-full-frame/route.ts

```diff
@@ -29,7 +29,7 @@ export async function POST(request: Request) {
 			portrait: portraitImage,
 			sendToDb: sendToDb === "true",
 		},
-		request.url,
```

Comment:
Small nit, is there a reason some extra commas were removed? I'm not going to comment on every one of these

Reply:
My Prettier did that, is it out of line with the project settings?